algo/src/test/java/com/hartwig/serve/sources/ckb/CkbTestFactory.java

```diff
@@ -117,6 +117,7 @@ public static ClinicalTrial createTrial(@NotNull String recruitment, @NotNull Li
                 .variantRequirement(Strings.EMPTY)
                 .variantRequirementDetails(variantRequirementDetails)
                 .locations(locations)
+                .ageGroups(ageGroups)
```

Comment:
Would suggest to stick to the same ordering as in the datamodel, in which case you should set `ageGroups` after `recruitment`